source/vsts.md

```diff
@@ -9,11 +9,24 @@ The [Azure Devops guides](https://g.octopushq.com/GuidesAzureDevops) provide ste
 
 ## Requirements
 
+## Step versions >=6.0.0
+
```

Comment:
Can we add the 2022.3 requirement here(**v6 of this step requires Octopus 2022.3+**), for people who skim through the first few lines of the documentation.